Program.cs

```diff
@@ -58,26 +67,14 @@ public static List<Dog> GetDogs()
         public static List<Cat> GetCats()
         {
             Cat diesel = new Cat("DIESEL", new DateTime(2008, 01, 01), "Archangielska");
-            diesel.Clinic = kaczor;
+            diesel.ClinicName = listOfClinics[1].Name;
 
             List<Cat> listaKotow = new List<Cat>();

```

Comment:
You don't have to declare and initial again listaKotow becouse You have it in line 21. Similarly with listOfClinics and list of dogs, just use them here.